### PR TITLE
Allow GCP buckets in other regions

### PR DESCRIPTION
**Pull request by KastenMike**
This PR lets you create buckets in specific regions or a different multi_regional than US. Currently default args are used (StorageClass: "MULTI_REGIONAL", Location: "US"); Location default is mentioned [here](https://pkg.go.dev/cloud.google.com/go/storage@v0.38.0#BucketAttrs), StorageClass through testing. Settings are passed via config like S3 does with `ConfigRegion`.

Example test (placed inside google/stow_test in my case):

```
func TestRegionalBucket(t *testing.T) {
	fmt.Println("TestRegionalBucket......")
	projectId := "projectId goes here"
	configJSON := "configJSON goes here"
	bucket := "mimckayd-deltest"
	storageClass := "REGIONAL"
	location := "asia-south2"
	cm := stow.ConfigMap{}
	cm[ConfigJSON] = configJSON
	cm[ConfigProjectId] = projectId
	cm[ConfigScopes] = ""
	cm[ConfigLocation] = location
	cm[ConfigStorageClass] = storageClass
	l, err := stow.Dial(Kind, cm)
	if err != nil {
		t.Errorf("NO DIAL! %v", err)
	}
	fmt.Println("dialled...")
	container, err := l.CreateContainer(bucket)
	if err != nil {
		t.Errorf("NO BUCKET! %v", err)
	}
	fmt.Println("bucket created...", container.Name())
}
```

If you comment out ConfigLocation/ConfigStorageClass settings from the test then it defaults to US+MULTI_REGIONAL as in the 2nd test here so the default behaviour remains the same:
image

Location/StorageClass aren't required to delete a bucket etc + `stow.Walk` still works as expected even without Location details passed (bucket is enough):
```
func TestDeleteBucket(t *testing.T) {
	fmt.Println("TestDeleteBucket......")
	projectId := "projectId goes here"
	configJSON := "configJSON goes here"
	bucket := "mimckayd-deltest"
	cm := stow.ConfigMap{}
	cm[ConfigJSON] = configJSON
	cm[ConfigProjectId] = projectId
	cm[ConfigScopes] = ""
	l, err := stow.Dial(Kind, cm)
	if err != nil {
		t.Errorf("NO DIAL! %v", err)
	}
	fmt.Println("dialled...")
	err = l.RemoveContainer(bucket)
	if err != nil {
		t.Errorf("NO BUCKET DELETION! %v", err)
	}
	fmt.Println("bucket deleted...")
}
```

### CODE REVIEW

File: google/location.go [hunk 1]
```diff
@@ -29,8 +29,15 @@ func (l *Location) Close() error {
 func (l *Location) CreateContainer(containerName string) (stow.Container, error) {
 
 	projId, _ := l.config.Config(ConfigProjectId)
+	location, _ := l.config.Config(ConfigLocation)
+	storageClass, _ := l.config.Config(ConfigStorageClass)
+
 	// Create a bucket.
-	_, err := l.client.Buckets.Insert(projId, &storage.Bucket{Name: containerName}).Do()
```
Comment (KastenMike) [inline google/location.go:33]: This code is quite out of date (1+ year) vs graymeta, maybe we should update but it can be a different PR https://github.com/graymeta/stow/pull/248

**Review by miquella — COMMENTED** [review]
I'm not sure that I know the `stow` APIs well enough to provide useful feedback here, but I will note that it looks like these files could use a `go fmt`

**Comment by KastenMike** [comment]
> looks like these files could use a `go fmt`

Fixed formatting

**Review by pavannd1 — APPROVED** [review]
